adobe_premiere_plugin/host/index.jsx

```diff
@@ -0,0 +1,8 @@
+function importFile(filePath) {
+  app.project.importFiles(
+    [filePath],
+    true, // suppress warnings
+    app.project.getInsertionBin(),
```

Comment:
What's an InsertionBin? Is this essentially selecting the internal library to import the file to?

Reply:
no idea! This was copy/pasta. But the docs tell us:

> Returns a ProjectItem object referencing the bin into which import will occur.

https://ppro-scripting.docsforadobe.dev/general/project.html#project-getinsertionbin